auditlog/models/rule.py

```diff
@@ -0,0 +1,677 @@
+            for deleted_id in deleted_ids:
+                old_value_text.append((deleted_id, "DELETED"))
+            vals["old_value_text"] = old_value_text
+            new_value_text = (
+                self.env[field["relation"]].browse(vals["new_value"]).name_get()
+            )
+            vals["new_value_text"] = new_value_text
+        return vals
+
+    def _create_log_line_on_create(self, log, fields_list, new_values):
+        """Log field filled on a 'create' operation."""
+        log_line_model = self.env["auditlog.log.line"]
+        for field_name in fields_list:
+            if field_name in FIELDS_BLACKLIST:
+                continue
+            field = self._get_field(log.model_id, field_name)
+            # not all fields have an ir.models.field entry (ie. related fields)
+            if field:
+                log_vals = self._prepare_log_line_vals_on_create(log, field, new_values)
+                log_line_model.create(log_vals)
+
+    def _prepare_log_line_vals_on_create(self, log, field, new_values):
+        """Prepare the dictionary of values used to create a log line on a
+        'create' operation.
+        """
+        vals = {
+            "field_id": field["id"],
+            "log_id": log.id,
+            "old_value": False,
+            "old_value_text": False,
+            "new_value": new_values[log.res_id][field["name"]],
+            "new_value_text": new_values[log.res_id][field["name"]],
+        }
+        if log.log_type == "full" and field["relation"] and "2many" in field["ttype"]:
+            new_value_text = (
+                self.env[field["relation"]].browse(vals["new_value"]).name_get()
+            )
+            vals["new_value_text"] = new_value_text
+        return vals
+
+    def subscribe(self):
+        """Subscribe Rule for auditing changes on model and apply shortcut
+        to view logs on that model.
+        """
+        act_window_model = self.env["ir.actions.act_window"]
+        for rule in self:
+            # Create a shortcut to view logs
+            domain = "[('model_id', '=', %s), ('res_id', '=', active_id)]" % (
+                rule.model_id.id
+            )
+            vals = {
+                "name": _("View logs"),
+                "res_model": "auditlog.log",
+                "binding_model_id": rule.model_id.id,
+                "domain": domain,
+            }
+            act_window = act_window_model.sudo().create(vals)
+            rule.write({"state": "subscribed", "action_id": act_window.id})
+        return True
+
+    def unsubscribe(self):
+        """Unsubscribe Auditing Rule on model."""
+        # Revert patched methods
+        self._revert_methods()
+        for rule in self:
+            # Remove the shortcut to view logs
+            act_window = rule.action_id
+            if act_window:
+                act_window.unlink()
+        self.write({"state": "draft"})
+        return True
```

Comment:
write returns `True` anyway.
```suggestion
        return self.write({"state": "draft"})
```